packages/suite/src/views/guide/Feedback.tsx

```diff
@@ -0,0 +1,298 @@
+    padding: 10px;
+    width: 100%;
+    margin: 0 0 20px;
+    cursor: pointer;
+`;
+
+const SelectWrapper = styled.div`
+    padding: 0 0 20px 0;
+`;
+
+const TextareaWrapper = styled.div`
+    position: relative;
+`;
+
+const FeedbackInfoNotice = styled.small`
+    display: block;
+    font-size: 10px;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    text-align: center;
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    padding: 0 0 8px;
+    width: 100%;
+`;
+
+const CharacterCount = styled.div`
+    position: absolute;
+    bottom: 35px;
+    right: 15px;
+    font-size: ${variables.FONT_SIZE.TINY};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+`;
+
+const RatingWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+    padding: 0 0 20px;
+`;
+
+const RatingItem = styled.button<{ selected?: boolean }>`
+    width: 48px;
+    height: 47px;
+    padding-top: 1px;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid ${props => props.theme.STROKE_GREY};
+    cursor: pointer;
+    font-size: 30px;
+
+    &:hover {
+        background: ${props => props.theme.STROKE_GREY};
+    }
+
+    ${props =>
+        props.selected &&
+        css`
+            background: ${props.theme.BG_GREEN};
+            border: 1px solid ${props.theme.BG_GREEN};
+
+            &:hover {
+                background: ${props.theme.BG_GREEN};
+            }
+        `};
+`;
+
+type RatingItem = {
+    id: Rating;
+    value: React.ReactNode;
+};
+
+interface Props {
+    type: FeedbackType;
+}
+
+const MESSAGE_CHARACTER_LIMIT = 1000;
+const ratingOptions: RatingItem[] = [
+    {
+        id: 1,
+        value: <>&#128545;</>,
+    },
+    {
+        id: 2,
+        value: <>&#128533;</>,
+    },
+    {
+        id: 3,
+        value: <>&#128529;</>,
+    },
+    {
+        id: 4,
+        value: <>&#128522;</>,
+    },
+    {
+        id: 5,
+        value: <>&#128525;</>,
+    },
+];
+
+const Feedback = ({ type }: Props) => {
+    const { device } = useDevice();
+    const { setView } = useActions({
+        setView: guideActions.setView,
+    });
+    const { sendFeedback } = useActions({
+        sendFeedback: guideActions.sendFeedback,
+    });
+    const [description, setDescription] = React.useState('');
+    const [rating, setRating] = React.useState<RatingItem>();
+    const [category, setCategory] = React.useState<Category>();
+
+    useEffect(() => {
+        if (description.length >= MESSAGE_CHARACTER_LIMIT) {
+            setDescription(description.slice(0, MESSAGE_CHARACTER_LIMIT));
+        }
+    }, [description]);
+
+    let firmwareType = '';
+    if (device) {
+        firmwareType = isBitcoinOnly(device) ? 'bitcoin-only' : 'regular';
+    }
+
+    const onSubmit = useCallback(() => {
+        const userData: UserDataCommon = {
+            platform: isDesktop() ? 'desktop' : 'web',
+            os: navigator.platform,
```

Comment:
```suggestion
            os: getPlatform(),
```
or better `getOsName()`